openfisca_us/variables/usda/snap/income/deductions/deductions.py

```diff
@@ -0,0 +1,26 @@
+from openfisca_us.model_api import *
+
+
+class snap_deductions(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "SNAP income deductions"
+    unit = "currency-USD"
+    documentation = "Deductions made from gross income for SNAP benefits"
+    definition_period = YEAR
+    reference = "United States Code, Title 7, Section 2014(e)"
+
+    def formula(spm_unit, period, parameters):
+        return sum(
```

Comment:
Would this work?
```python
DEDUCTIONS = ["standard", "earnings", "dependent_care", "child_support", "medical_expense", "shelter"]
return add(spm_unit, period, ["snap_" + i + "_deduction" for i in DEDUCTIONS])
```

Reply:
How about a list parameter?

Reply:
In general seems like we should use list parameters instead of list constants